caiman/summary_images.py

```diff
@@ -694,20 +694,30 @@ def local_correlations_movie(file_name,
 
 def local_correlations_movie_offline(file_name,
```

Comment:
Add a docstring with variable explanations. This should enter the documentation especially if it's used in demos.